front/components/spaces/AddToSpaceDialog.tsx

```diff
@@ -153,14 +155,20 @@ export const AddToSpaceDialog = ({
               variant="outline"
             />
           </DropdownMenuTrigger>
-          <DropdownMenuContent>
-            {availableSpaces.map((currentSpace) => (
-              <DropdownMenuItem
-                key={currentSpace.sId}
-                label={currentSpace.name}
-                onClick={() => setSpace(currentSpace)}
-              />
-            ))}
+          <DropdownMenuContent className="min-w-[200px] p-2">
+            <ScrollArea
+              className="flex max-h-[300px] flex-col overflow-y-auto"
```

Comment:
Let's avoid arbitrary values please 🙏. You can use either `h-72` or `h-80`.

Reply:
Couldn't figure out why `min-w-*` classes weren't available. But modified the tailwindconfig and it should work now